### DISCUSSION TITLE
The headlessui.com website is so dark

### DISCUSSION THREAD

**Discussion by xxnuo** [tailwindlabs/headlessui#3529] [Help]
The website is so dark, it's hard to read the text.

**Comment by xxnuo** [comment]
https://headlessui.com/

It's straining my eyes so bad 😭, please switch to a light theme

**Comment by jacogasp** [comment]
I agree, it is really difficult to read the examples in full daylight.
From a tailwind labs product, I would expect a good support of light/dark theme switching 😊
